falcon/settings.py

```diff
@@ -29,7 +26,6 @@
 
 ALLOWED_HOSTS = []
```

Comment:
can you please add explanation what is this field?

Reply:
I think that was auto-generated by django with the project creation. 
This line appears in one of the repo initial commits.

Reply:
yep, it's auto-generated.
more details about it you can find in:
https://docs.djangoproject.com/en/3.2/ref/settings/#allowed-hosts